app/__main__.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+"""Raw image data previewer - terminal functionality."""
+
+import argparse
+import os
+from .core import run_core_functionality
+
+parser = argparse.ArgumentParser(
+    prog=__package__,
+    description="preview raw data as an image of chosen format")
+parser.add_argument("FILE_PATH", help="file containing raw image data")
+parser.add_argument("-c",
+                    "--color_format",
+                    choices=["RGB3", "BGR3"],
+                    default="RGB3",
+                    help="target color format (default: %(default)s)")
+parser.add_argument("-r",
+                    "--resolution",
+                    metavar=("width", "height"),
+                    type=int,
+                    nargs=2,
+                    default=[600, 600],
+                    help="target resolution (default: %(default)s)")
+parser.add_argument("-p",
+                    "--parse",
+                    action='store_true',
+                    help="parse raw image data")
+parser.add_argument("-d",
+                    "--display",
+                    action='store_true',
+                    help="display image")
```

Comment:
I don't think we need this options. Parsed data (for now) is only used to display image and displaying image is default thing that will happen after starting CLI and providing file.